### PR TITLE
(SIMP-5170) Align Puppetfile versions with notes

### PR DESCRIPTION
**Pull request by trevor-vaughan**
This bumps the versions of ntpd and simp_elasticsearch to match what was
specified in
https://simp-project.atlassian.net/wiki/spaces/SD/pages/315195398/SIMP+6.2.0

That did pull in some very minor bug fixes but both appeared worthwhile
and properly tested.

SIMP-5170 #comment align Puppetfile

### CODE REVIEW

File: Changelog.rst [hunk 1]
```diff
@@ -138,6 +140,9 @@ pupmod-simp-aide
 ^^^^^^^^^^^^^^^^
 
 * Added /etc/logrotate.simp.d to default rules.
+* Ensure that the ``package`` install comes before dependent ``exec``
+  statements.
+* Allow the ``cron`` command to be customized.
```
Comment (lnemsick-simp) [inline Changelog.rst]: The update to allow cron to be customized hasn't completely been done (SIMP-5085).  So, this should be more explicit

File: Changelog.rst [hunk 2]
```diff
@@ -613,7 +656,12 @@ pupmod-simp-useradd
 simp-core
 ^^^^^^^^^
 
-* Add logic to auto.cfg to use OS-specific GPG keys in simp_filesystem.repo
+* Add logic to auto.cfg to use OS-specific GPG keys in simp_filesystem.repo.
```
Comment (lnemsick-simp) [inline Changelog.rst]: Fixes for UEFI ISO boot and UEFI PXEboot need to be noted as well.

File: Changelog.rst [hunk 3]
```diff
@@ -613,7 +656,12 @@ pupmod-simp-useradd
...
+
+simp-doc
+^^^^^^^^
+
+* Updated the documentation for UEFI PXE booting.
```
Comment (lnemsick-simp) [inline Changelog.rst]: Missing:

- Added HOWTO to IPA enrollment workflow
- Added documentation on how to disconnect from puppetDB if needed
- Updated contributors guide to describe additional details about development workflow
- Added FAQ clarifying different scenarios for managing SSH
- Removed OBE  MCollective references
- Added SIMP on AWS documentation

File: Changelog.rst (hunk 1 above)
Comment (lnemsick-simp) [reply Changelog.rst]: NB this is OK.  I thought this was about pupmod-simp-logrotate, not pupmod-simp-aide

**Comment by trevor-vaughan** [comment]
@lnemsick-simp If you want to go ahead and fix these things, that would be great.

**Pull request merged**